docs/reference/project/sites.mdx

```diff
@@ -2,18 +2,18 @@ Nitric sites allow static site content to be automatically uploaded to a bucket
 
```

Comment:
I think this first paragraph could use a refresh, with less assumed knowledge. What are static sites (what do they achieve/what are their goals/value, not what tech). Why/when would you use them? Then, how they're implemented in Nitric (as buckets, etc.)

Reply:
Updated let me know what you think